### DISCUSSION TITLE
Conditional disable resolver

### DISCUSSION THREAD

**Discussion by Malin88** [react-hook-form/react-hook-form#10864] [Q&A]
Hi, I have a question... is there any way to dynamically enable and disable the resolver in a form?
I have a zodResolver that validates form fields for me, but I would like this validation to take place only after pressing the "submit" button. In addition, I have the "auto save" function in the background, which calls me the onSubmit action every 10 seconds and sends data to the database, but in this case I would not like the form to be validated. Is there any way to do that?

**Comment by rub1e** [comment]
@Malin88 have you found a solution? 

I also want to be able to save partial form data without validating it, with validation only taking place on submission.

**Reply by lucien-theron** [reply]
I recently found I also need similar ability to conditionally disable or switch resolvers
